lib/shopify_cli/theme/syncer/json_update_handler.rb

```diff
@@ -86,6 +86,10 @@ def delete_locally(file)
         def ask_update_strategy(file)
           Forms::SelectUpdateStrategy.ask(@ctx, [], file: file, exists_remotely: file_exist_remotely?(file)).strategy
         end
+
+        def ready_to_update(file)
```

Comment:
I believe we could include the `?` suffix that boolean methods generally have.

Also, I'm wondering if we should call the method `ready_to_update?` or `updatable?`, as the ready may suggest we're waiting for something. Wdyt?